eclass/kernel-2.eclass

```diff
@@ -932,26 +931,26 @@ install_headers() {
 	# of this crap anymore :D
 	if kernel_is ge 2 6 18 ; then
 		env_setup_xmakeopts
-		emake headers_install INSTALL_HDR_PATH="${ED}"${ddir}/.. ${xmakeopts} || die
+		emake headers_install INSTALL_HDR_PATH="${ED%/}"${ddir}/.. ${xmakeopts} || die
```

Comment:
Is ${ddir} assumed to have a leading slash ?

Reply:
I was wondering that too :)

Reply:
Yes, `ddir` is assigned to the output of `kernel_header_destdir`, which always outputs a leading slash: https://github.com/gentoo/gentoo/blob/master/eclass/kernel-2.eclass#L692-L696